app/config/services/system.xml

```diff
@@ -271,5 +271,17 @@
             <tag name="form.type" />
         </service>
 
+        <!-- Security -->
```

Comment:
They can be put in `services.xml` where everything is autowired and autoconfigured by default

Reply:
done